src/librustc_passes/liveness.rs

```diff
@@ -843,6 +850,7 @@ impl<'a, 'tcx> Liveness<'a, 'tcx> {
 
             if changed {
                 this.rwu_table.assign_unpacked(idx, rwu);
+                any_changed = true;
```

Comment:
I guess this code was just writing back changes far too often before? (i.e., it also triggered if any previous index had changed)

Reply:
r=me if @nnethercote confirms this interpretation :)

Reply:
Yes, as per the commit message (which unfortunately isn't visible via the "Files changed" tab).